api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirObservationServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.api.impl;
+
+import javax.inject.Inject;
+
+import lombok.AccessLevel;
+import lombok.Setter;
+import org.hl7.fhir.r4.model.Observation;
+import org.openmrs.module.fhir2.api.FhirObservationService;
+import org.openmrs.module.fhir2.api.dao.FhirObservationDao;
+import org.openmrs.module.fhir2.api.translators.ObservationTranslator;
+import org.springframework.stereotype.Component;
+
+@Component
+@Setter(AccessLevel.PACKAGE)
+public class FhirObservationServiceImpl implements FhirObservationService {
+	
```

Comment:
should add `@Transactional` 
checkout [this](https://tickets.openmrs.org/browse/TRUNK-208)

Reply:
Thanks! I did not know that. I think instead of addressing this in this PR, I'm going to audit all of our ServiceImpl classes to ensure they are doing this.